schema/settings.schema.json

```
@@ -35,6 +35,12 @@
           "type": "boolean",
           "default": false,
           "!go": { "pointer": true }
+        },
+        "caseSensitiveSearch": {
+          "description": "Enables the search to be case sensitive. When true, patterns become case-sensitive (case-folding is disabled).",
```

```suggestion
          "description": "Enables the search to be case sensitive. When true, patterns become case-sensitive.",
```

Nit: the unfamiliar user probably doesn't know the term "case-folding", so probably better to just leave this out.